src/components/Stepper.tsx

```diff
@@ -0,0 +1,76 @@
+"use client";
+
+import React from "react";
+import { ArrowRightIcon } from "@heroicons/react/24/solid";
+import Button from "@/components/Button";
+import { cn } from "@/lib/utils";
+// TODO FOR LATER REFACTOR:
+//  - setting the width for the stepper: do we want to specify the width or base it off parent's width
+//  - variant colors for stepper button
+
+// chips or blocks
+type StepperStyle = "chips" | "icons";
+type StepperStatus = "completed" | "current" | "remaining";
+
+export interface SteppersItem {
+  icon?: string;
+  status: StepperStatus;
+  onClickEvent?: () => void;
+  name: string;
+}
+
+export interface StepperProps {
+  styleType: StepperStyle;
+  stepperWidth: string;
+  steppers: SteppersItem[];
+}
+
+export default function Stepper({
+  styleType,
+  stepperWidth,
+  steppers,
+}: StepperProps) {
+  return (
+    <div className={`${stepperWidth}`}>
+      <div className="flex">
+        {steppers.map((step) => {
+          const { icon, status, onClickEvent, name } = step;
+
+          let chipBgColor: string = "bg-neutral";
+          if (status === "current") chipBgColor = "bg-primary-content";
+          if (status === "completed") chipBgColor = "bg-success-content";
```

Comment:
I would prefer we can set these colors from outside the component as props

Reply:
colors will be hardcoded

remaining: neutral
completed: success-content
current: success